src/main/java/core/basesyntax/MatrixOperation.java

```diff
@@ -6,18 +6,42 @@
  */
 public class MatrixOperation {
     public int[] getDiagonal(int[][] matrix) {
-        return null;
+        int[] result = new int[matrix.length];
+        for (int i = 0, j = 0; i < matrix.length; i++, j++) {
```

Comment:
do we need `int j = 0`? try to do it without this variable

Reply:
you are right! I del j